doc/pydicom_dev_guide.rst

```diff
@@ -58,4 +58,11 @@ documentation of the ``master`` branch.
 PyPi build
 ----------
 
-TODO
+The package can be built for the different platform::
+  
+  python setup.py bdist_wininst
+  python setup.py sdist
+  python setup.py bdist_wheel --universal
```

Comment:
Just for my own reference:  I want to check into the wheel -- there was a [previous problem](https://github.com/pydicom/pydicom/issues/240#issuecomment-275588945) with this in making the `dicom` version -- not sure if it will apply to this version now that we can make a universal wheel.